package.json

```diff
@@ -19,7 +20,7 @@
     "debug": "^2.2.0",
     "lodash": "^3.9.3",
     "mpns": "^2.1.0",
-    "node-cache": "^2.2.0",
+    "node-cache": "^2.1.1",
```

Comment:
This look suspiciously to me, why are you downgrading the minimal required version of node-cache?